frontend/src/components/3.Preferences/Preferences.js

```diff
@@ -180,18 +193,15 @@ function Preferences() {
             onClick={() => setButtonPopup(true)}
             variant='info'
             size='lg'
-            id='HelpButton'
-          >
+            id='HelpButton'>
             Help
           </Button>
           <Help trigger={ButtonPopup} setTrigger={setButtonPopup}>
             <p>
-              Use this page to select a location, as well as a range from that
-              location to find restaurants from.<br></br>
-              Changing the timer duration will change how long you get to make a
-              decision on each restaurant.<br></br>
-              Use the price dropdown to set an upper price limit on the
-              restaurants that will be shown.<br></br>
+              Use this page to select a location, as well as a range from that location to find restaurants from.
+              <br></br>
```

Comment:
For this section, The text on the help button doesn't actually show the line break that is written into the code as in the previous version.

I think it was purposely written as 2 lines in the code to make it easier to read for the user, and to prevent lines of code that were too long.

I think either way is fine, so there is not need to change it back but a small nit is that on line 202, the `<br></br>` is on a new line, whereas 203 and 204 are inline.

Reply:
It might be on a new line because the line might have been too long with the text changes?

Reply:
This has now been fixed